code/go/0chain.net/chaincore/chain/handler.go

```diff
@@ -1943,8 +1943,13 @@ func SuggestedFeeHandler(ctx context.Context, r *http.Request) (interface{}, err
 
 	c := GetServerChain()
 	lfb := c.GetLatestFinalizedBlock()
+	if lfb == nil {
+		return nil, errors.New("LFB not ready yet")
+	}
+
+	lfb = lfb.Clone()
 
-	_, fee, err := c.EstimateTransactionCostFee(ctx, lfb.ClientState, &tx)
+	_, fee, err := c.EstimateTransactionCostFee(ctx, lfb, lfb.ClientState, &tx)
```

Comment:
if you pass lfb now, why do you need lfb.ClientState?

Reply:
Yes, lol, we don't need to pass the. `lfb.ClientState`, let me fix it.